File: html/tak.js
function tak(x, y, z) {
    if(x <= y) return z;
    return tak(tak(x - 1, y, z), tak(y - 1, z, x), tak(z - 1, x, y));
}
function c5(x) {
  return 5
}
function test(x, y, z) {
    var s = new Date().getTime();
    console.log(tak(x, y, z));
    var e = new Date().getTime();
    console.log(e - s);
}

File: vm/ncxi/ncxi_tmp/tak.c
int tak(int x, int y, int z) {
   if (y<x)
      return tak(tak(x-1,y,z),tak(y-1,z,x),tak(z-1,x,y));
   else
      return z;
};
int main(){
	int r = tak(20,10,0);
	return r;
};

